content/en/docs/contribute/review/reviewing.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Reviewing pull requests
+content_template: templates/concept
+main_menu: true
+weight: 10
+---
+
+{{% capture overview %}}
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/kubernetes/website/pulls) section in the Kubernetes website repository to see open pull requests.
+
+Reviewing documentation pull requests is a
+great way to introduce yourself to contributing to the Kubernetes community.
+It helps you learn the code base and build trust with other contributors.
+
+Before reviewing, it's a good idea to:
+
+- Read the  [content guide](/docs/contribute/style/content-guide/) and
+[style guide](/docs/contribute/style/style-guide/) so you can leave informed comments.
+- Understand the different [roles and responsibilities](/docs/contribute/participating/#roles-and-responsibilities) in the Kubernetes documentation community
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Before you begin
+
+Before you start a review:
+
+- Read the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and ensure that you abide by it at all times.
+- Be polite, considerate, and helpful.
+- Comment on positive aspects of PRs as well.
+- Be empathetic and mindful of how your review may be received.
+- Assume good intent and ask clarifying questions.
+- Experienced contributors, consider pairing with new contributors whose work requires extensive changes.
+
+## Review process
+
+In general, review pull requests for content and style in English.
```

Comment:
Line 46, Reviewers or readers of pull requests do not apply these labels (or at least I do not). Do you mean filter for PRs labelled with these strings?